### PR TITLE
slimserver: 7.9.2 -> 8.3.1

### LINKED ISSUE #178494: Slimserver media scan does not complete
### Describe the bug

Slimserver scans, but does not index all music files.

```
Media Scan Details
Discovering files/directories: /srv/music   (39480 of 39480)   Complete  00:00:04
Scanning new music files: /srv/music   (3724 of 3724)   Complete  00:00:04
Discovering playlists: /srv/music/playlists   (1 of 1)   Complete  00:00:00
Building full text index   (7 of 7)   Complete  00:00:00
Database Optimize   (2 of 2)   Complete  00:00:01
The server has finished scanning your media library.
```

### Steps To Reproduce
1. `services.slimserver.enable = true;`
2. rescan (clear library & rescan everything)
3. notice that many files are not indexed

### Expected behavior
All files to be indexed

### logs
Relevant parts of logs seem to be

scanner.log:
```
[22-06-21 21:16:25.7478] Slim::Utils::Scanner::Local::rescan (180) Discovering audio files in /srv/music
[22-06-21 21:16:25.7716] Slim::Utils::SQLiteHelper::updateProgress (442) Notify to server: [
  "progress:1655846185.7532||importer||/srv/music|discovering_directory||1||0||",
]
[22-06-21 21:16:25.8102] Slim::Utils::SQLiteHelper::updateProgress (471) Notify to server failed: 500 Server closed connection without sending any data back
[22-06-21 21:16:30.1206] Slim::Utils::SQLiteHelper::updateProgress (442) Notify to server: [
  "progress:1655846185||importer||/srv/music|discovering_directory||39480||39480||1655846190.12014",
]
[22-06-21 21:16:30.1221] Slim::Utils::SQLiteHelper::updateProgress (471) Notify to server failed: 500 Status read failed: Connection reset by peer
[22-06-21 21:16:30.1223] Slim::Utils::Scanner::Local::__ANON__ (192) Start processing found tracks
[22-06-21 21:16:30.1224] Slim::Utils::Scanner::Local::__ANON__ (200) Connect do DB

...


[22-06-21 21:16:38.7070] Slim::Music::PlaylistFolderScan::startScan (89) Starting playlist folder scan
[22-06-21 21:16:38.7072] Slim::Utils::Scanner::Local::rescan (180) Discovering audio files in /srv/music/playlists
[22-06-21 21:16:38.7165] Slim::Utils::SQLiteHelper::updateProgress (442) Notify to server: [
  "progress:1655846198.71578||importer||/srv/music/playlists|discovering_playlist||1||0||",
]
[22-06-21 21:16:38.7175] Slim::Utils::SQLiteHelper::updateProgress (471) Notify to server failed: 500 Status read failed: Connection reset by peer
[22-06-21 21:16:38.7180] Slim::Utils::SQLiteHelper::updateProgress (442) Notify to server: [
  "progress:1655846198||importer||/srv/music/playlists|discovering_playlist||1||1||1655846198.71777",
]
[22-06-21 21:16:38.7189] Slim::Utils::SQLiteHelper::updateProgress (471) Notify to server failed: 500 Server closed connection without sending any data back
[22-06-21 21:16:38.7191] Slim::Utils::Scanner::Local::__ANON__ (192) Start processing found tracks
[22-06-21 21:16:38.7192] Slim::Utils::Scanner::Local::__ANON__ (200) Connect do DB

...

[22-06-21 21:16:41.5876] Slim::Music::Import::runScanPostProcessing (443) Finished background scanning.
[22-06-21 21:16:41.5898] Slim::Utils::SQLiteHelper::updateProgress (442) Notify to server: ["end"]
[22-06-21 21:16:41.5908] Slim::Utils::SQLiteHelper::updateProgress (471) Notify to server failed: 500 Status read failed: Connection reset by peer
[22-06-21 21:16:41.5928] Slim::Utils::SQLiteHelper::updateProgress (442) Notify to server: ["exit"]
[22-06-21 21:16:41.5939] Slim::Utils::SQLiteHelper::updateProgress (471) Notify to server failed: 500 Status read failed: Connection reset by peer
```

(there are a lot of Connection reset by peer errors)

server.log:
```
...

[22-06-21 19:01:17.6568] Slim::Control::Request::execute (1888) Error: While trying to run function coderef [Slim::Control::Queries::statusQuery]: [Can't call method "can" without a package or object reference at /nix/store/h508k61gdg8vg7j6g8j8qcb63cxi98py-perl5.34.1-slimserver-8.2.0/Slim/Control/Queries.pm line 50
]
[22-06-21 19:01:17.6573] Slim::Web::JSONRPC::requestMethod (455) Request failed with error: Bad dispatch!
[22-06-21 19:01:17.6673] Slim::Control::Request::execute (1888) Error: While trying to run function coderef [Slim::Control::Queries::statusQuery]: [Can't call method "can" without a package or object reference at /nix/store/h508k61gdg8vg7j6g8j8qcb63cxi98py-perl5.34.1-slimserver-8.2.0/Slim/Control/Queries.pm line 50
]
[22-06-21 19:01:17.6677] Slim::Web::JSONRPC::requestMethod (455) Request failed with error: Bad dispatch!
[22-06-21 19:01:24.6700] Slim::Control::Request::execute (1888) Error: While trying to run function coderef [Slim::Control::Queries::statusQuery]: [Can't call method "can" without a package or object reference at /nix/store/h508k61gdg8vg7j6g8j8qcb63cxi98py-perl5.34.1-slimserver-8.2.0/Slim/Control/Queries.pm line 50
]
[22-06-21 19:01:24.6705] Slim::Web::JSONRPC::requestMethod (455) Request failed with error: Bad dispatch!
[22-06-21 19:01:24.6803] Slim::Control::Request::execute (1888) Error: While trying to run function coderef [Slim::Control::Queries::statusQuery]: [Can't call method "can" without a package or object reference at /nix/store/h508k61gdg8vg7j6g8j8qcb63cxi98py-perl5.34.1-slimserver-8.2.0/Slim/Control/Queries.pm line 50
]
[22-06-21 19:01:24.6807] Slim::Web::JSONRPC::requestMethod (455) Request failed with error: Bad dispatch!
[22-06-21 19:01:30.6804] Slim::Control::Request::execute (1888) Error: While trying to run function coderef [Slim::Control::Queries::statusQuery]: [Can't call method "can" without a package or object reference at /nix/store/h508k61gdg8vg7j6g8j8qcb63cxi98py-perl5.34.1-slimserver-8.2.0/Slim/Control/Queries.pm line 50
]
[22-06-21 19:01:30.6808] Slim::Web::JSONRPC::requestMethod (455) Request failed with error: Bad dispatch!
[22-06-21 19:01:30.6875] Slim::Control::Request::execute (1888) Error: While trying to run function coderef [Slim::Control::Queries::statusQuery]: [Can't call method "can" without a package or object reference at /nix/store/h508k61gdg8vg7j6g8j8qcb63cxi98py-perl5.34.1-slimserver-8.2.0/Slim/Control/Queries.pm line 50
]
[22-06-21 19:01:30.6877] Slim::Web::JSONRPC::requestMethod (455) Request failed with error: Bad dispatch!
[22-06-21 19:01:36.6930] Slim::Control::Request::execute (1888) Error: While trying to run function coderef [Slim::Control::Queries::statusQuery]: [Can't call method "can" without a package or object reference at /nix/store/h508k61gdg8vg7j6g8j8qcb63cxi98py-perl5.34.1-slimserver-8.2.0/Slim/Control/Queries.pm line 50
]
[22-06-21 19:01:36.6935] Slim::Web::JSONRPC::requestMethod (455) Request failed with error: Bad dispatch!
[22-06-21 19:01:36.7020] Slim::Control::Request::execute (1888) Error: While trying to run function coderef [Slim::Control::Queries::statusQuery]: [Can't call method "can" without a package or object reference at /nix/store/h508k61gdg8vg7j6g8j8qcb63cxi98py-perl5.34.1-slimserver-8.2.0/Slim/Control/Queries.pm line 50
]
[22-06-21 19:01:36.7024] Slim::Web::JSONRPC::requestMethod (455) Request failed with error: Bad dispatch!
[22-06-21 20:52:01.3224] main::init (387) Starting Logitech Media Server (v7.9.2, TRUNK, UNKNOWN) perl 5.034001 - x86_64-linux-thread-multi
[22-06-21 20:52:01.9748] Slim::Music::Import::addImporter (489) Adding Slim::Music::VirtualLibraries Scan
[22-06-21 20:52:01.9872] Slim::Music::Import::addImporter (489) Adding Slim::Media::MediaFolderScan Scan
[22-06-21 20:52:01.9874] Slim::Music::Import::countImporters (558) Found importer: Slim::Media::MediaFolderScan
[22-06-21 20:52:01.9898] Slim::Utils::SQLiteHelper::postConnect (377) Optimizing DB because of missing or empty sqlite_stat1 table
[22-06-21 20:52:01.9900] Slim::Schema::optimizeDB (398) Start schema_optimize
[22-06-21 20:52:01.9956] Slim::Schema::forceCommit (2143) Warning: Trying to commit transactions before DB is initialized!
[22-06-21 20:52:02.0139] Slim::Schema::optimizeDB (426) End schema_optimize
[22-06-21 20:52:02.1225] Slim::Utils::Misc::msg (1255) Warning: [20:52:02.1222] Constant subroutine Plugins::BBCSounds::ProtocolHandler::EWOULDBLOCK redefined at /nix/store/l2j4q3y94fh4xp32glsi70pas52n0i0d-perl5.34.1-Exporter-Lite-0.08/lib/perl5/site_perl/5.34.1/Exporter/Lite.pm line 56.
[22-06-21 20:52:02.1227] Slim::Utils::Misc::msg (1255) Warning: [20:52:02.1226] Constant subroutine Plugins::BBCSounds::ProtocolHandler::EINPROGRESS redefined at /nix/store/l2j4q3y94fh4xp32glsi70pas52n0i0d-perl5.34.1-Exporter-Lite-0.08/lib/perl5/site_perl/5.34.1/Exporter/Lite.pm line 56.
[22-06-21 20:52:02.1229] Slim::Utils::Misc::msg (1255) Warning: [20:52:02.1228] Constant subroutine Plugins::BBCSounds::ProtocolHandler::EINTR redefined at /nix/store/l2j4q3y94fh4xp32glsi70pas52n0i0d-perl5.34.1-Exporter-Lite-0.08/lib/perl5/site_perl/5.34.1/Exporter/Lite.pm line 56. 
[22-06-21 20:52:02.1231] Slim::Utils::Misc::msg (1255) Warning: [20:52:02.1229] Constant subroutine Plugins::BBCSounds::ProtocolHandler::ECHILD redefined at /nix/store/l2j4q3y94fh4xp32glsi70pas52n0i0d-perl5.34.1-Exporter-Lite-0.08/lib/perl5/site_perl/5.34.1/Exporter/Lite.pm line 56.
[22-06-21 20:52:02.1233] Slim::Utils::Misc::msg (1255) Warning: [20:52:02.1232] Constant subroutine Plugins::BBCSounds::ProtocolHandler::EBADF redefined at /nix/store/l2j4q3y94fh4xp32glsi70pas52n0i0d-perl5.34.1-Exporter-Lite-0.08/lib/perl5/site_perl/5.34.1/Exporter/Lite.pm line 56.
[22-06-21 20:52:02.1967] Slim::Plugin::ACLFiletest::Plugin::preinitPlugin (25) Error: Successfully initialized ACL filetests
[22-06-21 20:52:02.2001] Slim::Utils::PluginManager::load (392) Warning: Couldn't call Plugins::AutoRescan::Plugin->initPlugin: Can't locate object method "new" via package "Plugins::AutoRescan::Monitor_Linux" at /var/lib/slimserver/cache/InstalledPlugins/Plugins/AutoRescan/Plugin.pm line 111.
[22-06-21 20:52:02.2125] Slim::Music::Import::addImporter (489) Adding Slim::Plugin::FullTextSearch::Plugin Scan
[22-06-21 20:52:02.2127] Slim::Music::Import::countImporters (558) Found importer: Slim::Plugin::FullTextSearch::Plugin
[22-06-21 20:52:02.2128] Slim::Music::Import::countImporters (558) Found importer: Slim::Media::MediaFolderScan
[22-06-21 20:52:02.2232] Slim::Utils::SQLiteHelper::postConnect (377) Optimizing DB because of missing or empty sqlite_stat1 table
[22-06-21 20:52:02.2233] Slim::Schema::optimizeDB (398) Start schema_optimize
[22-06-21 20:52:02.2285] Slim::Schema::forceCommit (2143) Warning: Trying to commit transactions before DB is initialized!
[22-06-21 20:52:02.2450] Slim::Schema::optimizeDB (426) End schema_optimize
[22-06-21 20:52:02.3034] main::checkDataSource (1107) Warning: Schema updated or no media found in the database, initiating scan.
[22-06-21 20:52:02.3039] Slim::Music::Import::countImporters (558) Found importer: Slim::Plugin::FullTextSearch::Plugin
[22-06-21 20:52:02.3040] Slim::Music::Import::countImporters (558) Found importer: Slim::Media::MediaFolderScan
[22-06-21 20:52:20.4499] Slim::Schema::wipeCaches (2096) Wiped all in-memory caches.
[22-06-21 20:52:20.4511] Slim::Schema::wipeCaches (2096) Wiped all in-memory caches.
[22-06-21 20:52:20.5027] Slim::Schema::wipeCaches (2096) Wiped all in-memory caches.
[22-06-21 20:58:25.1644] main::init (387) Starting Logitech Media Server (v7.9.2, TRUNK, UNKNOWN) perl 5.034001 - x86_64-linux-thread-multi
[22-06-21 20:58:25.4338] Slim::Music::Import::addImporter (489) Adding Slim::Music::VirtualLibraries Scan
[22-06-21 20:58:25.4346] Slim::Music::Import::addImporter (489) Adding Slim::Media::MediaFolderScan Scan
[22-06-21 20:58:25.4348] Slim::Music::Import::countImporters (558) Found importer: Slim::Media::MediaFolderScan
[22-06-21 20:58:25.5466] Slim::Utils::Misc::msg (1255) Warning: [20:58:25.5464] Constant subroutine Plugins::BBCSounds::ProtocolHandler::EWOULDBLOCK redefined at /nix/store/l2j4q3y94fh4xp32glsi70pas52n0i0d-perl5.34.1-Exporter-Lite-0.08/lib/perl5/site_perl/5.34.1/Exporter/Lite.pm line 56.
[22-06-21 20:58:25.5469] Slim::Utils::Misc::msg (1255) Warning: [20:58:25.5467] Constant subroutine Plugins::BBCSounds::ProtocolHandler::EINPROGRESS redefined at /nix/store/l2j4q3y94fh4xp32glsi70pas52n0i0d-perl5.34.1-Exporter-Lite-0.08/lib/perl5/site_perl/5.34.1/Exporter/Lite.pm line 56.
[22-06-21 20:58:25.5470] Slim::Utils::Misc::msg (1255) Warning: [20:58:25.5469] Constant subroutine Plugins::BBCSounds::ProtocolHandler::EINTR redefined at /nix/store/l2j4q3y94fh4xp32glsi70pas52n0i0d-perl5.34.1-Exporter-Lite-0.08/lib/perl5/site_perl/5.34.1/Exporter/Lite.pm line 56.
[22-06-21 20:58:25.5472] Slim::Utils::Misc::msg (1255) Warning: [20:58:25.5471] Constant subroutine Plugins::BBCSounds::ProtocolHandler::ECHILD redefined at /nix/store/l2j4q3y94fh4xp32glsi70pas52n0i0d-perl5.34.1-Exporter-Lite-0.08/lib/perl5/site_perl/5.34.1/Exporter/Lite.pm line 56.
[22-06-21 20:58:25.5474] Slim::Utils::Misc::msg (1255) Warning: [20:58:25.5473] Constant subroutine Plugins::BBCSounds::ProtocolHandler::EBADF redefined at /nix/store/l2j4q3y94fh4xp32glsi70pas52n0i0d-perl5.34.1-Exporter-Lite-0.08/lib/perl5/site_perl/5.34.1/Exporter/Lite.pm line 56.
[22-06-21 20:58:25.6293] Slim::Plugin::ACLFiletest::Plugin::preinitPlugin (25) Error: Successfully initialized ACL filetests
[22-06-21 20:58:25.6322] Slim::Utils::PluginManager::load (392) Warning: Couldn't call Plugins::AutoRescan::Plugin->initPlugin: Can't locate object method "new" via package "Plugins::AutoRescan::Monitor_Linux" at /var/lib/slimserver/cache/InstalledPlugins/Plugins/AutoRescan/Plugin.pm line 111.
[22-06-21 20:58:25.6465] Slim::Music::Import::addImporter (489) Adding Slim::Plugin::FullTextSearch::Plugin Scan
[22-06-21 20:58:25.6466] Slim::Music::Import::countImporters (558) Found importer: Slim::Media::MediaFolderScan
[22-06-21 20:58:25.6468] Slim::Music::Import::countImporters (558) Found importer: Slim::Plugin::FullTextSearch::Plugin
[22-06-21 20:58:26.0465] Slim::Web::Cometd::handler (421) errorNeedsClient: b8:27:eb:5f:b5:ed, status, -, 10, menu:menu, useContextMenu:1, subscribe:600
[22-06-21 20:58:26.1744] Slim::Web::Cometd::handler (421) errorNeedsClient: 00:04:20:23:8c:7e, status, -, 10, menu:menu, useContextMenu:1, subscribe:600
[22-06-21 20:58:27.4716] Slim::Web::Cometd::handler (421) errorNeedsClient: b8:27:eb:b2:7f:d6, status, -, 10, menu:menu, useContextMenu:1, subscribe:600
[22-06-21 21:01:54.0685] Slim::Web::Settings::Server::Basic::handler (63) Initiating scan of type: wipecache
[22-06-21 21:01:54.0776] Slim::Music::Import::countImporters (558) Found importer: Slim::Media::MediaFolderScan
[22-06-21 21:01:54.0779] Slim::Music::Import::countImporters (558) Found importer: Slim::Plugin::FullTextSearch::Plugin

...
```

### Additional context
Discussion [on slimdevices forum.](https://forums.slimdevices.com/showthread.php?112147-Many-missing-tracks-and-album-when-scanning&p=1056615#post1056615)

### Things I have already tried

- I have also tried 7.9.3 and even got 8.2.0 working but with similar problems. I suspect it's a perl dependency issue, but I don't know.
- I tried overriding Audioscan with 0.9.3

### Notify maintainers
@phile314 

### Metadata
```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.17.15, NixOS, 22.05 (Quokka), 22.05.1043.5afb1b7dcf4`
 - multi-user?: `no`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.8.1`
 - channels(root): `"nixos-22.05"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

### PR DESCRIPTION
**Pull request by jecaro**
Slimserver aka LogitechMediaServer was broken (see #178494)

This PR brings it to the latest version making the package usable again. I'm currently using this on nixOS running on a RPI4 and it's been working fine for the last few weeks.

I added myself as a maintainer as I'd be happy to fix the package if any new issue arises.

###### Description of changes

- Fix #178494
- upstream [changlog](https://htmlpreview.github.io/?https://github.com/Logitech/slimserver/blob/4f40dde1fdb0cc38e361e0a28e599c8e42e54ecc/Changelog8.html)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by posch** [comment]
Thanks for this pull request. I've tested it with Squeezebox Boom and Squeezebox Touch, and it seems to work just fine.
FWIW, I think this should be merged.

**Comment by jecaro** [comment]
Gently pinging @phile314 

That PR has been sitting here for a few weeks now. It would be great if you could have a look soon. 

Thx in advance :pray:

**Comment by phile314** [comment]
As I am no longer using slimserver, I will remove myself as maintainer. I am of course more than happy if you want to take over :-)

If I have seen this correctly, this updates slimserver and uses the bundled CPAN copies from slimserver instead of the ones in nixpkgs. During the initial packaging using the bundled CPAN libraries caused trouble, but if it works nowadays that might be the easier path.

**Review by phile314 — APPROVED** [review]

File: pkgs/servers/slimserver/default.nix [hunk 1]
```diff
@@ -1,105 +1,43 @@
...
-    perlPackages.TimeDate
-    perlPackages.URI
-    perlPackages.URIFind
-    perlPackages.UUIDTiny
-    perlPackages.XMLParser
-    perlPackages.XMLSimple
-    perlPackages.YAMLLibYAML
-  ];
 
+  buildInputs = [ perl534Packages.IOSocketSSL ];
```
[the remark this answers is not in the document]
Comment (jecaro) [reply pkgs/servers/slimserver/default.nix:28]: Yes, I have read that issue when trying to package it using the modules shipped with NixOS. I found it super hard to make it works reliably that way.

It is not necessary to add a comment here as it is the way slimserver is packaged in other distros anyway (I've checked the ubuntu package and the archlinux AUR). So we just follow the recommended way to package it basically. Also, the change is clearly stated in the body of the commit message. So it'll be easily found by anybody digging into git history.

**Comment by jecaro** [comment]
The branch has been rebased to solve the conflict introduced by https://github.com/NixOS/nixpkgs/pull/222249

**Comment by adamcstephens** [comment]
Trying this out, but unable to install the Shairtunes2 plugin.

```
Mar 23 19:12:37 bank slimserver.pl[18459]: [23-03-23 19:12:36.6923] Slim::Utils::Misc::msg (1325) Warning: [19:12:36.6919] Can't locate auto/Crypt/OpenSSL/RSA/autosplit.ix in @INC (@INC contains: /data/lms/cache/InstalledPlugins /data/lms/cache/InstalledPlugins /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1/CPAN/arch/5.34/x86_64-linux-thread-multi /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1/CPAN/arch/5.34/x86_64-linux-thread-multi/auto /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1/CPAN/arch/5.34.1/x86_64-linux-thread-multi /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1/CPAN/arch/5.34.1/x86_64-linux-thread-multi/auto /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1/CPAN/arch/5.34/x86_64-linux-thread-multi /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1/CPAN/arch/5.34/x86_64-linux-thread-multi/auto /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1/CPAN/arch/x86_64-linux-thread-multi /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1/CPAN/arch/5.34 /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1/lib /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1/CPAN /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1 /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1 /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1/lib/perl5/site_perl /nix/store/hywj1knkyl64ijjyqmchxvgv6nz2kdc2-perl5.34.1-Net-SSLeay-1.92/lib/perl5/site_perl/5.34.1/x86_64-linux-thread-multi /nix/store/hywj1knkyl64ijjyqmchxvgv6nz2kdc2-perl5.34.1-Net-SSLeay-1.92/lib/perl5/site_perl/5.34.1 /nix/store/hywj1knkyl64ijjyqmchxvgv6nz2kdc2-perl5.34.1-Net-SSLeay-1.92/lib/perl5/site_perl /nix/store/89vnj4nzcg22z8v0lk4rjp3wa7xjl0ni-perl5.34.1-Mozilla-CA-20200520/lib/perl5/site_perl/5.34.1/x86_64-linux-thread-multi /nix/store/89vnj4nzcg22z8v0lk4rjp3wa7xjl0ni-perl5.34.1-Mozilla-CA-20200520/lib/perl5/site_perl/5.34.1 /nix/store/89vnj4nzcg22z8v0lk4rjp3wa7xjl0ni-perl5.34.1-Mozilla-CA-20200520/lib/perl5/site_perl /nix/store/ix4jiqhghb4qaf3aiinyvw96b8fh758y-perl5.34.1-IO-Socket-SSL-2.068/lib/perl5/site_perl/5.34.1/x86_64-linux-thread-multi /nix/store/ix4jiqhghb4qaf3aiinyvw96b8fh758y-perl5.34.1-IO-Socket-SSL-2.068/lib/perl5/site_perl/5.34.1 /nix/store/ix4jiqhghb4qaf3aiinyvw96b8fh758y-perl5.34.1-IO-Socket-SSL-2.068/lib/perl5/site_perl /nix/store/bgpn8pkjshbm03iy58m8g96x0aga3nci-perl-5.34.1/lib/perl5/site_perl/5.34.1/x86_64-linux-thread-multi /nix/store/bgpn8pkjshbm03iy58m8g96x0aga3nci-perl-5.34.1/lib/perl5/site_perl/5.34.1 /nix/store/bgpn8pkjshbm03iy58m8g96x0aga3nci-perl-5.34.1/lib/perl5/site_perl /nix/store/bgpn8pkjshbm03iy58m8g96x0aga3nci-perl-5.34.1/lib/perl5/site_perl -I/nix/store/ix4jiqhghb4qaf3aiinyvw96b8fh758y-perl5.34.1-IO-Socket-SSL-2.068/lib/perl5/site_perl -I/nix/store/ /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1/CPAN/arch/5.34/x86_64-linux-thread-multi /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1/CPAN/arch/5.34/x86_64-linux-thread-multi/auto /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1/CPAN/arch/5.34.1/x86_64-linux-thread-multi /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1/CPAN/arch/5.34.1/x86_64-linux-thread-multi/auto /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1/CPAN/arch/x86_64-linux-thread-multi /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1/CPAN/arch/5.34 /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1/lib /nix/store/6l9rvr8wvhmf1yf465cf4366dsndw3aq-perl5.34.1-slimserver-8.3.1/CPAN /nix/store/bgpn8pkjshbm03iy58m8g96x0aga3nci-perl-5.34.1/lib/perl5/5.34.1/x86_64-linux-thread-multi /nix/store/bgpn8pkjshbm03iy58m8g96x0aga3nci-perl-5.34.1/lib/perl5/5.34.1 /data/lms/cache/InstalledPlugins/Plugins/ShairTunes2W/elib /data/lms/cache/InstalledPlugins/Plugins/ShairTunes2W/elib/5.34 /data/lms/cache/InstalledPlugins/Plugins/ShairTunes2W/elib/5.34/x86_64-linux-thread-multi /data/lms/cache/InstalledPlugins/Plugins/ShairTunes2W/elib/5.34/x86_64-linux-thread-multi/auto) at /nix/store/bgpn8pkjshbm03iy58m8g96x0aga3nci-perl-5.34.1/lib/perl5/5.34.1/AutoLoader.pm line 181.
Mar 23 19:12:37 bank slimserver.pl[18459]:  at /data/lms/cache/InstalledPlugins/Plugins/ShairTunes2W/elib/Crypt/OpenSSL/RSA.pm line 11.
```

**Comment by jecaro** [comment]
I've added the package `perl534Packages.CryptOpenSSLRSA` in the `buildInputs`. Now the Shairtunes2 plugin seems to load fine. I don't know how to test that it works correctly thus. Do I need an Apple product for this? If so, unfortunately, I don't have one. So please try again @adamcstephens and lemme know how it goes.

As for the installed packages in the dockerfile,  `perl534Packages.CryptOpenSSLRSA` should indeed be the equivalent package to `libcrypt-openssl-rsa-perl` so I think we should be fine here. Thx for the link btw, that's a good hint of what we need to make that package works.

**Comment by adamcstephens** [comment]
> I don't know how to test that it works correctly thus. Do I need an Apple product for this? If so, unfortunately, I don't have one. So please try again @adamcstephens and lemme know how it goes.

There may be an open source sender, but I've never looked. I updated with your changes and am now able to successfully airplay from my iphone. :+1: Thanks!

**Comment by jecaro** [comment]
>  I'm happy to help maintain if you're open to it. Feel free to add me to the list, or I can do so later.

Sure np. Happy to have more maintainers for this. Doing it now.

**Review by adamcstephens — APPROVED** [review]

**Review by adamcstephens — APPROVED** [review]
approving again.

**Pull request merged**